app/lib/account_session.rb

```diff
@@ -103,7 +103,15 @@ def migrate_transition_checker_email_subscription
   attr_reader :session_signing_key
 
   def get_remote_attributes(remote_attributes)
-    remote_attributes.index_with { |name| oidc_do :get_attribute, { attribute: name } }.compact
+    values = remote_attributes.index_with do |name|
+      if userinfo.key? name
+        userinfo[name]
+      else
+        oidc_do(:get_attribute, { attribute: name })
```

Comment:
nitpick - feel free to ignore. Looks like we usually leave off the brackets in the rest of this class when calling `oidc_do`